synapse/storage/events.py

```diff
@@ -293,19 +366,19 @@ def _persist_event_txn(self, txn, event, context, backfilled,
                 txn, event.event_id, context.rejected
             )
 
-        for hash_alg, hash_base64 in event.hashes.items():
-            hash_bytes = decode_base64(hash_base64)
-            self._store_event_content_hash_txn(
-                txn, event.event_id, hash_alg, hash_bytes,
-            )
+        # for hash_alg, hash_base64 in event.hashes.items():
+        #     hash_bytes = decode_base64(hash_base64)
+        #     self._store_event_content_hash_txn(
+        #         txn, event.event_id, hash_alg, hash_bytes,
+        #     )
```

Comment:
Ooooh, an interesting question. I don't think those tables actually get read from currently? So the question is whether we still want to keep them?